File: 250326/시험 통과 여부 확인하기 2/verify-test-passed-2.py
n=int(input())
total=0

cnt=0
for _ in range(n):
    arr=list(map(int,input().split()))
    for elem in arr:
        total+=elem
    if total/len(arr) >= 60:
        print("pass")
        cnt+=1
           
    else:
        print("fail")
        
    total=0
    arr=[]
print(cnt)

File: 250326/시험 통과 여부 확인하기 2/verify-test-passed-2.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    
    
    int cnt=0;
    for(int i=0; i<n; i++){
        int arr[4];
        int total=0;
        for(int i=0; i<4; i++){
            cin >> arr[i];
            total+=arr[i];

        } 
        if(total/4 >= 60){
            cout << "pass" << endl;
            cnt++;
        }
        else{
            cout << "fail" << endl;
        }
        
    }
      
        
    
    cout << cnt;

    // Please write your code here.
    return 0;
}

